cpp/src/arrow/sparse_tensor.h

```diff
@@ -136,50 +139,80 @@ class ARROW_EXPORT SparseCOOIndex : public internal::SparseIndexBase<SparseCOOIn
   std::shared_ptr<Tensor> coords_;
 };
 
-// ----------------------------------------------------------------------
-// SparseCSRIndex class
+namespace internal {
 
-/// \brief EXPERIMENTAL: The index data for a CSR sparse matrix
-///
-/// A CSR sparse index manages the location of its non-zero values by two
-/// vectors.
-///
-/// The first vector, called indptr, represents the range of the rows; the i-th
-/// row spans from indptr[i] to indptr[i+1] in the corresponding value vector.
-/// So the length of an indptr vector is the number of rows + 1.
-///
-/// The other vector, called indices, represents the column indices of the
-/// corresponding non-zero values.  So the length of an indices vector is same
-/// as the number of non-zero-values.
-class ARROW_EXPORT SparseCSRIndex : public internal::SparseIndexBase<SparseCSRIndex> {
- public:
-  static constexpr SparseTensorFormat::type format_id = SparseTensorFormat::CSR;
+struct SparseMatrixCompressedAxis {
```

Comment:
Can make this a `enum class` instead of an enum-in-a-struct.

Reply:
Changed.